apps/blockchain/README.md

```diff
@@ -1,4 +1,4 @@
-# Mana-Ethereum Blockchain [![CircleCI](https://circleci.com/gh/exthereum/blockchain.svg?style=svg)](https://circleci.com/gh/exthereum/blockchain)
```

Comment:
I'm guessing we're removing this here since we have the badge in the main README?

Reply:
yep